src/GafferSceneUI/StandardLightVisualiser.cpp

```diff
@@ -464,6 +464,15 @@ Visualisations StandardLightVisualiser::visualise( const IECore::InternedString
 	// frustum group ignores local, and inherits visualiser scale.
 	GroupPtr frustum = new Group;
 
+	addWireframeCurveState( ornaments.get() );
+	addConstantShader( ornaments.get() );
+	addWireframeCurveState( geometry.get() );
+	addConstantShader( geometry.get() );
+	addWireframeCurveState( boundOrnaments.get() );
+	addConstantShader( boundOrnaments.get() );
+	addWireframeCurveState( frustum.get() );
+	addConstantShader( frustum.get() );
```

Comment:
Aren't we better off doing this at the end of the function, for only the things that ended up with something in them?